doc/api/top-level.rst

```diff
@@ -0,0 +1,30 @@
+.. currentmodule:: xarray
+
+Top-level functions
+===================
+
+.. autosummary::
+   :toctree: ../generated/
+
+   apply_ufunc
+   align
```

Comment:
Could add subsections here similar to DataArray  ("Configuration", "Computation", etc.)